coeus-impl/src/main/java/org/kuali/coeus/common/budget/impl/personnel/BudgetPersonValuesFinder.java

```diff
@@ -94,6 +94,15 @@ private String getPersonUniqueKey(BudgetPerson budgetPerson) {
     	uniqueKey.append(budgetPerson.getEffectiveDate());
     	return uniqueKey.toString();
     }
+    
+    protected String getBudgetPersonLabel(BudgetPerson budgetPerson) {
+    	StringBuffer personLabel = new StringBuffer();
+    	personLabel.append(budgetPerson.getPersonName());
+    	personLabel.append(" (");
+    	personLabel.append(budgetPerson.getJobCode());
```

Comment:
I tested all scenario, it did not break. Will check for null. Currently it will indicate that the job code is null.